src/components/network/network-panel.js

```diff
@@ -181,5 +219,36 @@ genotet.NetworkPanel.prototype.tooltipNode = function(node) {
 genotet.NetworkPanel.prototype.tooltipEdge = function(edge) {
   var tooltip = genotet.tooltip.create();
   this.setEdgeInfo_(edge, tooltip);
-  tooltip.find('.close').remove();
+  tooltip.find('#rm-edge, .close').remove();
+};
+
+/**
+ * Hides the node-info when removing nodes.
+ * @param {!Array<string>} genes Genes to hide.
+ */
+genotet.NetworkPanel.prototype.hideNodeInfo = function(genes) {
+  var info = this.container.find('#node-info');
+  var name = info.children('#name').children('span').text();
+  var geneMap = genotet.utils.keySet(genes);
+  if (name in geneMap) {
+    this.hideNodeInfo_();
+  }
+};
+
+/**
+ * Hides the edge-info when removing edges.
+ * @param {!Array<!genotet.NetworkEdge>} edges Edges to hide.
+ */
+genotet.NetworkPanel.prototype.hideEdgeInfo = function(edges) {
+  var info = this.container.find('#edge-info');
+  var source = info.children('#source').children('span').text();
+  var target = info.children('#target').children('span').text();
+  var edgeId = source + ',' + target;
+  var edgeIdMap = {};
+  edges.forEach(function(edge) {
+      edgeIdMap[edge.id] = true;
+  });
+  if (edgeId in edgeIdMap) {
```

Comment:
You are also only searching once here. You can either edges.map ids and then indexOf, or you can for loop the edges and terminate immediately when the searched edge id is found.